TESTS/netsocket/README.md

```diff
@@ -1068,527 +1088,513 @@ Test whether timeouts are obeyed in TCPSockets.
 
 **Postconditions:**
 
-1.  Call `TCPSocket::close();`.
-2.  Delete socket.
+1. Call `TCPSocket::close();`.
+1. Delete the socket.
 
 **Expected result:**
 
-`TCPSocket::getsockopt(keepalive)` returns same value as was set with `TCPSocket::setsockopt()` or `NSAPI_ERROR_UNSUPPORTED'.
+`TCPSocket::getsockopt(keepalive)` returns the same value as that set with `TCPSocket::setsockopt()` or `NSAPI_ERROR_UNSUPPORTED`.
 
 Test cases for TLSSocket class
 ------------------------------
 
 ### TLSSOCKET_OPEN_DESTRUCT
 
-**Description:** Run�SOCKET_OPEN_DESTRUCT for TLSSocket.
+**Description:** Run `SOCKET_OPEN_DESTRUCT` for TLSSocket.
 
 ### TLSSOCKET_OPEN_LIMIT
 
-**Description:** Run�SOCKET_OPEN_LIMIT for TLSSocket.
+**Description:** Run `SOCKET_OPEN_LIMIT` for TLSSocket.
 
 ### TLSSOCKET_OPEN_TWICE
 
-**Description:** Run�SOCKET_OPEN_TWICE�for TLSSocket.
+**Description:** Run `SOCKET_OPEN_TWICE` for TLSSocket.
 
 ### TLSSOCKET_CONNECT_INVALID
 
-**Description:** Run�SOCKET_CONNECT_INVALID for TLSSocket.
+**Description:** Run `SOCKET_CONNECT_INVALID` for TLSSocket.
 
 ### TLSSOCKET_HANDSHAKE_INVALID
 
 **Description:**
 
-Execute TLS handshake by calling `TLSSocket::connect()` - server must not match to the certificate used by to os.mbed.com
+Execute the TLS handshake by calling `TLSSocket::connect()`. The server must not match the certificate used by os.mbed.com.
 
 **Preconditions:**
 
-1.  Network interface and stack are initialized.
-2.  Network connection is up.
+1. Network interface and stack are initialized.
+1. Network connection is up.
 
 **Test steps:**
 
-1.  Create TLSSocket.
-2.  Call `TLSSocket::open()`.
-3.  Call `TLSSocket::connect("os.mbed.com", 2009)`.
-4.  Call `TLSSocket::close()`.
+1. Create TLSSocket.
+1. Call `TLSSocket::open()`.
+1. Call `TLSSocket::connect("os.mbed.com", 2009)`.
+1. Call `TLSSocket::close()`.
 
 **Expected result:**
 
-TLSSocket::connect must return an error
+`TLSSocket::connect` must return an error.
 
 ### TLSSOCKET_SEND_CLOSED
 
 **Description:**
 
-Make a HTTP request to a closed socket.
+Make an HTTP request to a closed socket.
 
 **Preconditions:**
 
-1.  Network interface and stack are initialized.
-2.  Network connection is up.
+1. Network interface and stack are initialized.
+1. Network connection is up.
 
 **Test steps:**
 
-1.  Create TLSSocket.
-2.  Call `TLSSocket::open()`.
-3.  Call `TLSSocket::connect("echo.mbedcloudtesting.com", 2007)`.
-4.  Call `TLSSocket::close()`.
-5.  Call `TLSSocket::send("12345", 5)`.
+1. Create TLSSocket.
+1. Call `TLSSocket::open()`.
+1. Call `TLSSocket::connect("echo.mbedcloudtesting.com", 2007)`.
+1. Call `TLSSocket::close()`.
+1. Call `TLSSocket::send("12345", 5)`.
 
 **Expected result:**
 
-TLSSocket::send must return an error
+`TLSSocket::send` must return an error.
 
 ### TLSSOCKET_SEND_REPEAT
 
-**Description:** Run SOCKET_SEND_REPEAT for TLSSOCKET by using port number 2009.
+**Description:** Run `SOCKET_SEND_REPEATÂ` for TLSSOCKET by using port number 2009.
```

Comment:
It looks like there's some weird formatting going on. Do we know what this symbol should be, or should we delete it?

Reply:
Sorry but i don't know that.